core/src/main/scala/chisel3/dontTouch.scala

```diff
@@ -34,6 +36,8 @@ object dontTouch {
   def apply[T <: Data](data: T): T = {
     requireIsHardware(data, "Data marked dontTouch")
     data match {
+      case d if DataMirror.hasProbeTypeModifier(d) => ()
+      case _:   Property[_] => ()
```

Comment:
Explicitly returning `Unit`, i.e. `()`, is probably best here. Most places in the code base will not make this explicit and just put no text after the `=>`. I like the style here of `()`, though.

Reply:
👍 i agree that i like explicitly returning Unit

Reply:
(Oh, neat! TIL, thanks!)